integration-test.sh

```diff
@@ -35,4 +35,4 @@ fi
 echo "Running integration tests..."
 make out/executor
 pushd integration
-go test -v --bucket "${GCS_BUCKET}" --repo "${IMAGE_REPO}" --timeout 30m
+go test -v --bucket "${GCS_BUCKET}" --repo "${IMAGE_REPO}" --timeout 30m -run ^TestRun/Dockerfile_test_add$
```

Comment:
I think this may be left over from testing?

Reply:
D'oh. Thanks!